app/models/spree/affirm_checkout.rb

```diff
@@ -3,8 +3,6 @@ class AffirmCheckout < ActiveRecord::Base
     belongs_to :payment_method
     belongs_to :order
 
-    validate :validate_checkout_matches_order
```

Comment:
Is there another validation?

Reply:
We have the standard price validation in another price. This validation was technically removed a while back, I'm just cleaning up unused functions.